FRW/App/AppDelegate.swift

```diff
@@ -20,27 +20,32 @@ import WalletCore
 import Web3Wallet
 
 #if DEBUG
-    import Atlantis
+import Atlantis
```

Comment:
Could we make a separate PR for the formatting and lint ? 
There are so many changes are lint and formatting changes, it's impossible to review other changes